Minimal changes required to fix demo bug

jmhewitt opened the pull request:
Hi dcurrier, I just wanted to share that I just started playing with carousel panels in shiny and came across your demo script.  I also came across another recent solution for the same by smbache at https://gist.github.com/smbache/eeef4bf21b053da780eb.  There's one extra piece of the carousel that smbache included in the other demo... I'm not sure what this extra code does to session$clientData, but I added it to your carouselPanel and it makes the slide visibility work nicely for what I was interested in seeing.

Thanks for sharing your carouselPanel code!  

-Josh

dcurrier commented:
Hi Josh,
Good find!  I have not been developing anything with Shiny lately, but this is a very helpful update.  I merged your edits in and moved the javascript from `carousel_events()` to a `tags$script()` in the main function.  
One thing I did notice in testing was that this requires Shiny version 0.12.   On shiny version 0.10 the second plot did not render until I advanced the carousel AWAY from it.  I updated to version 0.12 and it all worked the way I think it should.

Thanks for the suggestion,
Duane

jmhewitt commented:
Excellent and interesting.  Happy to help.

Thanks,
Josh

On Tue, Sep 8, 2015 at 2:23 PM, dcurrier notifications@github.com wrote:

> Hi Josh,
> Good find! I have not been developing anything with Shiny lately, but this
> is a very helpful update. I merged your edits in and moved the javascript
> from carousel_events() to a tags$script() in the main function.
> 
> One thing I did notice in testing was that this requires Shiny version
> 0.12. On shiny version 0.10 the second plot did not render until I advanced
> the carousel AWAY from it. I updated to version 0.12 and it all worked the
> way I think it should.
> 
> Thanks for the suggestion,
> Duane
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dcurrier/carouselPanel/pull/1#issuecomment-138688719.
